From: Oswald Buddenhagen <ossi@kde.org>
Date: Sat, 05 Mar 2011 18:39:00 +0000
Subject: default to poweroff for HaltCmd on linux
X-Git-Url: http://quickgit.kde.org/?p=kde-workspace.git&amp;a=commitdiff&amp;h=a58c0a17217b14e2e47415cfdfb1e8bb0b61a4d3
---
default to poweroff for HaltCmd on linux

now that linux can have more bsd-like shutdown implementations (both in
form of gentoo and of systemd), it makes sense to treat it the same.

BUG: 267490
FIXED-IN: 4.6.2
---


--- a/kdm/config.def
+++ b/kdm/config.def
@@ -37,7 +37,7 @@
 #ifdef _AIX
 # define HALT_CMD	"/usr/sbin/shutdown -h now"
 # define REBOOT_CMD	"/usr/sbin/shutdown -r now"
-#elif defined(BSD)
+#elif defined(BSD) || defined(__linux__)
 # define HALT_CMD	"/sbin/shutdown -p now"
 # define REBOOT_CMD	"/sbin/shutdown -r now"
 #elif defined(__SVR4)

